docs/components/collider.md

```diff
@@ -16,28 +16,80 @@ Be aware that the event will be emitted by the `RigidBody` parent
-| **activeCollision**      | To set the collider receiver/emitter collision events                                                                    | `false`                            |
-| **activeCollisionTypes** | Type of the collision event.                                                                    | `ActiveCollisionTypes.DEFAULT`                            |
-| **collisionGroups**      | To specify collision groups.                                                                    | `undefined`                            |
-
-:::info
-You can access the [Collider](https://rapier.rs/docs/user_guides/javascript/colliders) instance
-which offers full control over all the properties & methods available
-by using [Template refs](https://vuejs.org/guide/essentials/template-refs.html#template-refs).
-:::
+| Prop                     | Description                  | Default |
+| :----------------------- | :--------------------------- | :------- |
+| **shape**                | shape of the collider | `cuboid` |
+| **args**                 | The half-sizes of the collider shapes | `[1,1,1]` |
+| **object**               | Required for certain shapes like `trimesh`, `hull`, `heightfield`. | |
+| **friction**             | The friction coefficient of this collider. (automatic-collider) | `0.5` |
+| **mass**                 | Mass of the collider. (automatic-collider) | `1` |
+| **density**              | Restitution controls how elastic (aka. bouncy) a contact is. (automatic-collider) | `0` |
+| **restitution**          | The collider density. If non-zero the collider's mass and angular inertia will be added. (automatic-collider). | `1` |
+| **activeCollision**      | To set the collider receiver/emitter collision events | `false` |
+| **activeCollisionTypes** | Type of the collision event. | `ActiveCollisionTypes.DEFAULT` |
+| **collisionGroups**      | To specify collision groups. | `undefined` |
+| **sensor**      | Set the collider as senor. More details [here](#sensor). | `undefined` |
 
-## Expose object
+## Events
+
+The `Collider` component comes with a set of useful events allowing actions based on collisions or intersections (aka sensor).
+
+### Sensor
+
+The **Sensor** feature allows events to be triggered when there's an intersection or in other words, when the collider is traversed by another collider.
+
+The traversed `Collider` (or the collider that will trigger events), is the sensor and should set the `activeCollision` and `sensor` properties to `true`.
+By passing the above properties, the collider will no longer be affected by the physics law and will now start triggering the following intersection events:
+
+- **@intersection-enter**:  When another collider starts to traverse the *sensor*
+- **@intersection-exit**: When another collider leave the *sensor*
+
+> ℹ️ Note that you can directly pass those properties to the **`RigidBody`** for **auto-colliders**.

```

Comment:
@Neosoulink You can use an info block here for this note. Similar to the one below